### PR TITLE
Feature/137 findings

### PR DESCRIPTION
**Pull request by timohuber**

### CODE REVIEW

**Comment by timohuber** [comment]
- custom fields: labels of validations
- email: layout is tenant/root specific

File: pool/app/email/helper.ml [hunk 1]
```diff
@@ -20,7 +20,46 @@ let prepend_root_directory pool url =
   | false -> url
 ;;
 
-let prepare_email pool language label subject email params =
+let layout_from_tenant (tenant : Pool_tenant.t) =
+  let open Pool_tenant in
+  let tenant_url = tenant.url in
```
Comment (timohuber) [inline pool/app/email/helper.ml:25]: unnecessary

File: pool/app/email/helper.ml [hunk 2]
```diff
@@ -20,7 +20,46 @@ let prepend_root_directory pool url =
...
+  let logo_alt = tenant.title |> Title.value |> Format.asprintf "Logo %s" in
+  let link = tenant_url |> Url.value |> Format.asprintf "http://%s" in
+  { link; logo_src; logo_alt }
+;;
+
+let root_layout () =
+  let open CCOption in
+  let root_url =
+    Sihl.Configuration.read_string "PUBLIC_URL"
+    >>= fun url -> url |> Pool_tenant.Url.create |> CCOption.of_result
```
Comment (timohuber) [inline pool/app/email/helper.ml:43]: CCFun. %>

File: pool/app/custom_field/entity.ml [hunk 3]
```diff
@@ -254,6 +254,16 @@ module Validation = struct
     let all = [ number_min, `Number; number_max, `Number ]
   end
 
+  let key_to_human key =
```
[the remark this answers is not in the document]
Comment (aronerben) [reply pool/app/custom_field/entity.ml:297]: Done

File: pool/app/email/entity.ml [hunk 4]
```diff
@@ -14,6 +14,14 @@ module Sihl_email = struct
   ;;
 end
 
+(* TODO: Add name to use instead of Pool Tool *)
```
[the remark this answers is not in the document]
Comment (aronerben) [reply pool/app/email/entity.ml:17]: Discussed in person, will be done later

**Review by aronerben — APPROVED** [review]

**Pull request merged**